extensions/wikia/AdEngine/js/slot/inContent.js

```diff
@@ -1,36 +1,26 @@
 /*global define*/
 define('ext.wikia.adEngine.slot.inContent', [
 	'ext.wikia.adEngine.adTracker',
+	'ext.wikia.adEngine.context.slotsContext',
 	'JSMessages',
+	'wikia.document',
 	'wikia.log',
 	'wikia.window'
-], function (adTracker, msg, log, win) {
+], function (adTracker, slotsContext, msg, doc, log, win) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Function has 6 parameters which is greater than 4 authorized. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AExcessiveParameterList)